docs/backups.md

```diff
@@ -19,28 +21,56 @@ You can store backups outside of Kubernetes cluster in one of the supported clou
 
 ![image](assets/images/backup-cloud.svg)
 
-If you're running a Kubernetes cluster on premises, you can  store backups inside it using a [Persistent Volume :octicons-link-external-16:](https://kubernetes.io/docs/concepts/storage/persistent-volumes/). For example, if you don't use a remote backup storage or if storage costs are high for you. 
+If you're running a Kubernetes cluster on premises, you can  store backups inside it using a [Persistent Volume :octicons-link-external-16:](https://kubernetes.io/docs/concepts/storage/persistent-volumes/). For example, if you don't use a remote backup storage or if storage costs are high for you.
 
 ![image](assets/images/backup-pv.svg)
 
-## Workflow
+## Backup methods
+
+The Operator supports two backup methods:
 
-After you create a Backup object, the Operator sets up a backup Pod that runs Percona XtraBackup inside and create a backup Job. It also creates a path in the storage to save the backup data.
+### SST method (default)
 
-The backup Job within the backup Pod starts copying the data files from the Percona XtraDB Cluster to the backup storage. The Percona XtraDB Cluster Pod that serves the data enters the Donor state and stops receiving all requests.
+This backup method uses State Snapshot Transfer (SST). When you create a Backup object, the Operator:
+
+1. Sets up a backup Pod that runs Percona XtraBackup inside and creates a backup Job
+2. Creates a path in the storage to save the backup data
+3. Starts copying the data files from the Percona XtraDB Cluster to the backup storage
+4. The Percona XtraDB Cluster Pod that serves the data enters the Donor state and stops receiving all requests
 
 The backup task is resource-consuming and can affect performance. That's why the Operator uses one of the secondary Percona XtraDB Cluster Pods for backups. The exception is a one-pod deployment, where the same Pod is used for all tasks.
 
 After the data files are copied, the Operator marks the backup Pod as 'Completed' and deletes it. The Operator also updates the status of the Backup object.
 
+### XtraBackup sidecar method
+
+When you enable the `XtrabackupSidecar` feature gate, the Operator uses a different backup approach:
+
+1. An XtraBackup sidecar container runs in each Percona XtraDB Cluster Pod, providing an HTTP server interface
+2. When you create a Backup object, the Operator sends backup requests directly to the sidecar
+3. The sidecar performs the backup and uploads it directly to cloud storage (S3, Azure, or GCP)
+
+**Benefits of the XtraBackup sidecar method:**
```

Comment:
another benefit of sidecar method: SST method was hard to troubleshoot in case something goes wrong.